Sources/LiveViewNative/Utils/ShapeStyle/AnyShapeStyle.swift

```diff
@@ -201,21 +201,23 @@ extension AnyShapeStyle: Decodable {
             self = Self(BackgroundStyle())
         }
         
-        var modifiers = try container.nestedUnkeyedContainer(forKey: .modifiers)
-        while !modifiers.isAtEnd {
-            let modifier = try modifiers.nestedContainer(keyedBy: CodingKeys.Modifier.self)
-            switch try modifier.decode(Modifier.self, forKey: .type) {
-            case .opacity:
-                self = Self(self.opacity(try modifier.decode(Double.self, forKey: .properties)))
-            case .blendMode:
-                self = Self(self.blendMode(try modifier.decode(BlendMode.self, forKey: .properties)))
-            case .shadow:
-                self = Self(self.shadow(try modifier.decode(ShadowStyle.self, forKey: .properties)))
+        if container.contains(.modifiers) {
```

Comment:
This check shouldn't be needed, the Elixir side always sets the `modifiers`. Is there some scenario where that fails?

Reply:
It's not working with the templates in the Scratchpad demo. I gave more context in the PR description up top and there is also a [Slack thread](https://dockyard.slack.com/archives/C02E1GA5THB/p1683067516382179).

Reply:
I can't reproduce this. Could it have been a version mismatch between the client package and the Elixir package?

Reply:
@carson-katri I updated the `main` branch of the [scratchboard](https://github.com/liveview-native/scratchboard) repo which should allow you to easily reproduce this. Here's how it currently looks, using the `main` branch of liveview-client-swiftui as a package dependency in the Xcode project:

![Screenshot 2023-05-03 at 6 33 37 PM](https://user-images.githubusercontent.com/5893007/236090920-2eddf253-0aed-49a7-bb26-acbfde713cd4.png)

The changes in this branch seem to fix it (you'll have to rebase `anyshape-keys` off `main` if you're testing, as it doesn't include your template syntax changes from [this PR](https://github.com/liveview-native/liveview-client-swiftui/pull/787) which scratchboard has been updated to use)

Reply:
I removed the check for the modifiers key. I was having issues with package mismatches in my local environment so maybe it was a me problem.

@supernintendo Does this PR for your as it is now? If so then I'll update the description and we can merge it.

Reply:
Everything is working fine for me in my environment after May updated the packages. Sorry for the false alarm. I'll merge this PR with only the CodingKey changes.